packages/widget_toolkit/README.md

```diff
@@ -19,11 +19,11 @@ The Widget Toolkit package consists of several lightweight and customisable UI c
 
 | Item Picker (mutli-select)| Item Picker (single select)| Language Picker |
 |---------------------------------------|-----------------------------|-----------------------------|
-| <img src="https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/packages/widget_toolkit/master/doc/assets/multi-item-picker.gif" alt="Address Widget"> | <img src="https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/master/doc/assets/single-item-picker.gif" alt="Language Picker"></img> | <img src="https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/master/doc/assets/language-picker.gif" alt="Language Picker"></img> |
+| <img src="https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/packages/widget_toolkit/master/doc/assets/multi-item-picker.gif" alt="Address Widget"> | <img src="https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/packages/widget_toolkit/master/doc/assets/single-item-picker.gif" alt="Language Picker"></img> | <img src="https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/packages/widget_toolkit/master/doc/assets/language-picker.gif" alt="Language Picker"></img> |
```

Comment:
I believe `Prime-Holding/widget_toolkit/packages/widget_toolkit/master` needs to be replaced by `Prime-Holding/widget_toolkit/master/packages/widget_toolkit`